test/EmbedIO.Tests/BasicAuthenticationModuleTest.cs

```diff
@@ -39,13 +40,15 @@ public async Task RequestWithInvalidCredentials_ReturnsUnauthorized()
 
             var response = await MakeRequest(UserName, wrongPassword).ConfigureAwait(false);
             Assert.AreEqual(HttpStatusCode.Unauthorized, response.StatusCode, "Status Code Unauthorized");
+            Assert.AreEqual("Basic realm=\"/\" charset=UTF-8", response.Headers.WwwAuthenticate.ToString());
```

Comment:
It's a good thing you added this check, but it should be in its own method, maybe named `RequestWithInvalidCredentials_ReturnsValidWWWAuthenticateHeader`.